src/lib/crawler/handler.ts

```diff
@@ -1,29 +1,45 @@
 import { Context, SQSEvent } from "aws-lambda";
-//* import { WarmaneCrawler } from "./crawler";
+import { WarmaneCrawler } from "./crawler";
+import DynamoService from "../../dynamoService";
 // import bodyParser from "koa-bodyparser";
 
+const dynamoService = new DynamoService("MatchData");
+
 export async function crawlerHandler(
   event: SQSEvent,
-  context: Context
-  //* warmaneCrawler: WarmaneCrawler
+  context: Context,
+  warmaneCrawler: WarmaneCrawler
```

Comment:
I don't think that warmaneCralwer should not be a dependency to this thing. You can just new it up within the invocation of this function.